sites/all/modules/custom/fj16_job_ads/fj16_job_ads.install

```diff
@@ -46,6 +46,13 @@ function fj16_job_ads_update_7008() {
   fj16_import_translation('fj16_job_ads', 'sv');
 }
 
+function fj16_job_ads_update_9000() {
```

Comment:
Tän pitäis olla fj16_job_ads_update_7013, kats. dokumentaatio: https://api.drupal.org/api/drupal/modules!system!system.api.php/function/hook_update_N/7